docs/release-notes/next/GH-3192-cors-headers.md

```diff
@@ -0,0 +1 @@
+* Added CORS headers for error responses
```

Comment:
I could be nitpicking here. Let's merge this. But in the future when writing these, consider the reader has no context. Doesn't know the change. So I would wonder, responses from what API.